src/app/cli/src/coda_worker_testnet.ml

```diff
@@ -226,7 +226,7 @@ let start_payment_check logger root_pipe payment_pipe workers testnet
       let user_commands_pass_inspection =
         Hashtbl.fold user_commands_under_inspection ~init:[]
           ~f:(fun ~key:user_cmd
-             ~data:{snapshots; passed_root}
+             ~data:{snapshots; passed_root; _}
```

Comment:
Since the tests pass, can you refactor the code to use the refactoring that I recommended the other day (using priority queue to determine if a payment made it within an expected deadline)? It would make this function easier to maintain for other people in the long run.

Reply:
Sure.